src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -107,6 +108,10 @@ export const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
       template = <ScreeningRoom venue={venue} />;
       break;
 
+    case VenueTemplate.meetingroom:
```

Comment:
[NOTE]
I think this `TemplateWrapper` should incrementally be deconstructed by removing concerns out of it and ultimately replaced by a simple `Record<VenueTemplate, ReactNode>` in `VenuePage.tsx`